Cannot resolve module 'react-dom/lib/CSSProperty'

Getting this error after installing 0.5.2:

```
ERROR in ./~/draft-js-export-html/lib/helpers/styleToCSS.js
Module not found: Error: Cannot resolve module 'react-dom/lib/CSSProperty' in /Users/dmt0/projects/web/node_modules/draft-js-export-html/lib/helpers
 @ ./~/draft-js-export-html/lib/helpers/styleToCSS.js 7:19-55
```

--- a/src/helpers/styleToCSS.js
+++ b/src/helpers/styleToCSS.js
@@ -1,12 +1,59 @@
 /* @flow */
 
-import {isUnitlessNumber} from 'react-dom/lib/CSSProperty';
-
 type StyleDescr = {[key: string]: number | string};
 
 const VENDOR_PREFIX = /^(moz|ms|o|webkit)-/;
 const NUMERIC_STRING = /^\d+$/;
 const UPPERCASE_PATTERN = /([A-Z])/g;
+
+// Lifted from:
+// https://github.com/facebook/react/blob/ab4ddf64939aebbbc8d31be1022efd56e834c95c/src/renderers/dom/shared/CSSProperty.js
+const isUnitlessNumber = {
+    animationIterationCount: true,
+    borderImageOutset: true,
+    borderImageSlice: true,
+    borderImageWidth: true,
+    boxFlex: true,
+    boxFlexGroup: true,
+    boxOrdinalGroup: true,
+    columnCount: true,
+    flex: true,
+    flexGrow: true,
+    flexPositive: true,
+    flexShrink: true,
+    flexNegative: true,
+    flexOrder: true,
+    gridRow: true,
+    gridRowEnd: true,
+    gridRowSpan: true,
+    gridRowStart: true,
+    gridColumn: true,
+    gridColumnEnd: true,
+    gridColumnSpan: true,
+    gridColumnStart: true,
+    fontWeight: true,
+    lineClamp: true,
+    lineHeight: true,
+    opacity: true,
+    order: true,
+    orphans: true,
+    tabSize: true,
+    widows: true,
+    zIndex: true,
+    zoom: true,
+
+    // SVG-related properties
+    fillOpacity: true,
+    floodOpacity: true,
+    stopOpacity: true,
+    strokeDasharray: true,
+    strokeDashoffset: true,
+    strokeMiterlimit: true,
+    strokeOpacity: true,
+    strokeWidth: true
+};
+
+
 
 // Lifted from: https://github.com/facebook/react/blob/master/src/renderers/dom/shared/CSSPropertyOperations.js
 function processStyleName(name: string): string {

